.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.12"
+  - "5.9.1"
```

Comment:
is there a reason for 5.9 ? I get than 0.12 is old, but 4.x will be maintained to 2018, leaving us time to adapt, where 5.x is no longer supported in a month.